apps/meteor/ee/app/license/server/settings.ts

```diff
@@ -20,4 +20,11 @@ await settingsRegistry.addGroup('Enterprise', async function () {
 			i18nLabel: 'Status',
 		});
 	});
+	await this.add('Cloud_Workspace_AirGapped_Restrictions_Remaining_Days', -1, {
+		type: 'int',
+		hidden: true,
+		readonly: true,
+		secret: true,
```

Comment:
This secret property is only used in the frontend to use password inputs instead of regular inputs. I don't think it is necessary aswell.